guest/api/helper/helpers.go

```diff
@@ -0,0 +1,16 @@
+package helper
+
+import (
+	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/api"
+	protoapi "sigs.k8s.io/kube-scheduler-wasm-extension/kubernetes/proto/api"
+)
+
+// TolerationsTolerateTaint checks if taint is tolerated by any of the tolerations.
+func TolerationsTolerateTaint(tolerations []*protoapi.Toleration, taint *protoapi.Taint) bool {
```

Comment:
This refers to this function https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/toleration.go#L38-L57.
As the `kubernetes` folder is sparsely checked out and I guess should not be modified, is there a better approach how to use this function?

Reply:
You should put it (along with `ToleratesTaint(...)`) inside examples/nodeunschedulable.
I don't think we should maintain this function as the part of the guest SDK.